packages/testlab/package.json

```diff
@@ -20,11 +20,14 @@
   "copyright.owner": "IBM Corp.",
   "license": "MIT",
   "dependencies": {
+    "@loopback/core": "^4.0.0-alpha.27",
```

Comment:
I find this rather suspicious.

 - `testlab` must not depend on any other packages, otherwise we get circular dependencies
 - AFAICT, `@loopback/core` is not used anywhere in testlab.

Can you revert (remove) this line please?

Reply:
Done. That was accidental.